draft/0005-impersonation.md

```diff
@@ -242,14 +239,16 @@ Content-Type: application/json
 The resource owner API validates the support engineer's request and call Logto's Management API using a machine-to-machine application to issue a subject token for the user.
 
 ```
-POST /api/security/subject-tokens HTTP/1.1
```

Comment:
Seems the API still has the `security/` in the path? Should we remove this part?

Reply:
The `security` prefix has been removed in the master.